sh.exe.stackdump

```diff
@@ -0,0 +1,28 @@
+Exception: STATUS_STACK_OVERFLOW at rip=7FF891154D87
```

Comment:
Has this file been committed by accident?

Reply:
I think so. I don't know why it keeps coming up. It's not the first time.

Reply:
Do I delete it?

Reply:
Yes please.

If you do a `git rm` on it and then commit that it should get rid of it. It's a good idea to check what you are going to commit with `git status` before you do the final commit. Hopefully you'll spot things like this that shouldn't be included.

Reply:
:+1: